### PR TITLE
v2.0.0

### PR DESCRIPTION
**Pull request by ekaterina-nbl**
- Added JSON formatted output for all the commands (fixed https://github.com/electricimp/imp-central-impt/issues/16)
- Fixed security issue with the `sync-exec` library (just got rid of it)

### CODE REVIEW

File: lib/DeviceGroup.js [hunk 1]
```diff
@@ -414,25 +420,26 @@ class DeviceGroup extends Entity {
...
             catch(error => UserInteractor.processError(error));
     }
 
     _printDevicesList(devices, message, ...messageArgs) {
         return this._helper.makeConcurrentOperations(devices, (device) => device._collectShortListData()).
             then(() => {
-                UserInteractor.printMessage(message, ...messageArgs);
-                UserInteractor.printEntities(devices);
+                UserInteractor.printInfo(message, ...messageArgs);
+                UserInteractor.printJsonInfo(devices.map(entity => entity.displayData));
```
Comment (ppetrosh) [inline lib/DeviceGroup.js]: `printJson`, `printJsonInfo`, `printJsonDialog` look really confusing. It should be `printObj***` I believe instead

**Comment by ppetrosh** [comment]
`$ ./bin/impt auth login -h`

produces:

```
--output, -z     Adjusts the command output. (may be repeated several times)
                   (may be repeated several times)
                                   [array] [choices: "minimal", "json", "debug"]
```

**Comment by ppetrosh** [comment]
If that's not too hard at least for the JSON output I would make all the field names lower_case_underscore_separated (as everywhere).

**Comment by ppetrosh** [comment]
And also as discussed offline, I believe eventually we'll need to have `--format` (text, json) and `--level` (minimal, debug, etc) options. But I don't want to change anything about it now.

File: lib/DeviceGroup.js (hunk 1 above)
Comment (ekaterina-nbl) [reply lib/DeviceGroup.js]: Yes, you are right. Fixed

**Comment by ekaterina-nbl** [comment]
As discussed offline, multiple occurrence of --output option was prohibited; JSON field names are left unchanged.
All above issues are fixed.

**Pull request merged**